spec/requests/resources_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+require "digest"
+require "faraday"
```

Comment:
We no longer need the "faraday" and "digest" gems since we just use Rack::Test and did not end up checking MD5s of the files. So you can delete those lines.

Reply:
done